ocs_ci/templates/ocp-deployment/install-config-aws-ipi.yaml.j2

```
@@ -35,6 +35,9 @@ networking:
   networkType: OpenShiftSDN
   serviceNetwork:
     - 172.30.0.0/16
+{% if fips %}
+fips: {{ fips | default('false') }}
```

```suggestion
fips: {{ fips }}
```
no need for default value as if not defined we don't  add anything

---

ack. Will fix